02/app.js

```diff
@@ -1,2 +1,2 @@
 console.log('Działa! Gratuluję uruchomienia JS z poziomu Node.js!');
-console.log('Moja wersja Node.js to: ');
\ No newline at end of file
+console.log('Moja wersja Node.js to: 6.10.3');
```

Comment:
Wersja node.js jest bardzo niska. Na pewno wpisałaś poprawną wersję?
Jeśli tak to musisz ją zaktualizować ;)